src/test/resources/pages/instructorCommentsPageSearchNormal.html

```diff
@@ -126,7 +126,7 @@
                   </span>
                   .instructor1@course1.tmt at ${datetime.now})
                 </span>
-                <a class="btn btn-default btn-xs icon-button pull-right" data-original-title="Edit comment in the Comments page" data-placement="top" data-toggle="tooltip" href="/page/instructorCommentsPage?user=comments.idOfInstructor1OfCourse1&courseid=comments.idOfTypicalCourse1#${comment.id}" rel="noopener noreferrer" style="display:none;" target="_blank" title="" type="button">
+				<a class="btn btn-default btn-xs icon-button pull-right" data-original-title="Edit comment in the Comments page" data-placement="top" data-toggle="tooltip" href="/page/instructorCommentsPage?user=comments.idOfInstructor1OfCourse1&courseid=comments.idOfTypicalCourse1#${comment.id}" rel="noopener noreferrer" style="display:none;" target="_blank" title="" type="button">
```

Comment:
That's not nitpicking, its a valid concern! I can't believe I missed this even after checking each line for space instead of tabs... :(